charts/karmada-operator/values.yaml

```diff
@@ -58,7 +58,37 @@ operator:
     ##   - myRegistryKeySecretName
     ##
     pullSecrets: []
-  ## @param.resources
+
+  ## @param operator.env List of environment variables to inject
+  ##
+  ## - Each entry must be a valid Kubernetes EnvVar object.
+  ## - Supports both literal values and valueFrom references (ConfigMap, Secret, fieldRef, etc.).
+  ## - If omitted or set to an empty array (`[]`), no env stanza will be included.
+  ##
+  ## A sample stanza is shown below.
+  ##
+#  env:
+#    - name: HTTP_PROXY
+#      value: "http://best-awesome-proxy.com:8080"
+#    - name: http_proxy
+#      value: "http://best-awesome-proxy.com:8080"
+#    - name: HTTPS_PROXY
+#      value: "http://best-awesome-proxy.com:8080"
+#    - name: https_proxy
+#      value: "http://best-awesome-proxy.com:8080"
+#    - name: NO_PROXY
+#      value: "localhost,127.0.0.1,*.svc,*.cluster.local"
+#    - name: no_proxy
+#      value: "localhost,127.0.0.1,*.svc,*.cluster.local"
```

Comment:
If i remember correctly, it is not allowed to specify the same environment variable twice, one in uppercase and one in lowercase. 

My question is why we need to have them both in example. kind of misleading to me.

Reply:
Good question. It's possible to set both. There's some history as to why it may be useful to do that, but in this case that actually does not really matter, so I'll simplify this to only the lower case variants which should be honored by most *nix tools and  libraries.